module/dice/dice.mjs

```diff
@@ -112,12 +112,15 @@ export async function d20Roll({
  * @returns {{isFF: boolean, advantageMode: number}}  Whether the roll is fast-forward, and its advantage mode.
  */
 function _determineAdvantageMode({event, advantage=false, disadvantage=false, fastForward=false}={}) {
-  const isFF = fastForward || (event && (event.shiftKey || event.altKey || event.ctrlKey || event.metaKey));
+  const keys = {
+    normal: event ? _areKeysPressed(event, "d20RollFastForwardNormal") : false,
+    advantage: event ? _areKeysPressed(event, "d20RollFastForwardAdvantage") : false,
+    disadvantage: event ? _areKeysPressed(event, "d20RollFastForwardDisadvantage") : false
+  };
+  const isFF = fastForward || Object.values(keys).some(k => k);
```

Comment:
My module, Advantage Reminder, does this same kind of check to see if the user is fast-forwarding rolls. I basically copied the original way `isFF` was calculated. It's a bit more complicated now, which is to be expected. I'm wondering for my use case if I'd be better off just calling `_determineAdvantageMode` myself but my only hesitation is that it's marked as a private function (starting with `_`).

You alluded to potentially moving some of this into `util.js`. If that makes it more reusable by modules, like mine, then consider this a +1 to doing that.